### PR TITLE
Added RHEL 8 support

### PR DESCRIPTION
**Pull request by yogeshprasad**
Added RHEL 8 support

Jira ticket:  https://wavefront.atlassian.net/browse/INT-460
Reviewers: @vikramraman

### CODE REVIEW

**Comment by yogeshprasad** [comment]
@vikramraman 
Please take a look

File: sh/install.sh [hunk 1]
```diff
@@ -43,6 +44,39 @@ function exit_with_failure() {
...
+        then
+            yum install -y python >> ${INSTALL_LOG} 2>&1
+            if [ $? -eq 0 ]
+            then
+                    echo "Installed python"
+            else
+                    echo "Failed to install python"
+            fi
+        else
+            PYTHON_PATH=$(which python2)
```
Comment (vikramraman) [inline sh/install.sh]: python 2 will not be supported after the end of this year. Could we look into using python 3 instead?

File: sh/install.sh [hunk 2]
```diff
@@ -79,7 +113,16 @@ function install_python() {
         apt-get install python -y >> ${INSTALL_LOG} 2>&1
     elif [ $OPERATING_SYSTEM == "REDHAT" ]; then
         echo "Installing Python using yum"
-        yum install python -y >> ${INSTALL_LOG} 2>&1
+
+        file_name=/etc/redhat-release
+
+        # Check if file exists
+        if [ -f $file_name ]
```
Comment (vikramraman) [inline sh/install.sh]: Is `/etc/redhat-release` not there on newer versions of redhat? If it is always expected on a redhat machine, maybe we should just log a error message on the else clause and not attempt to install python.

File: sh/install.sh (hunk 1 above)
Comment (yogeshprasad) [reply sh/install.sh]: As of now wavefront CLI only supports python 2.
Is it fine to add the python 3 support for Wavefront CLI as a different task?

File: sh/install.sh (hunk 2 above)
Comment (yogeshprasad) [reply sh/install.sh]: I have updated the logic
/etc/redhat-release or /etc/system-release-cpe file will always be present

File: sh/install.sh (hunk 1 above)
Comment (vikramraman) [reply sh/install.sh]: @yogeshprasad Have you tried using python3? What is the failure?

The python2 EOL is coming very soon. We should try to understand what the issue with python3 is and fix it. Specifically installing python2 is not a good idea.

**Comment by yogeshprasad** [comment]
closing this as created new one https://github.com/wavefrontHQ/wavefront-cli/pull/21

**Pull request closed without merging**